docs/dasharo-tools-suite/documentation/features.md

```diff
@@ -474,6 +482,40 @@ version. This is how we can achieve that.
     version: 2022-08-31_cbff21b
     ```
 
+## Fusing the device vendor keys
+
+DTS can be used to fuse the device vendor keys onto the SoC to enable
+the Dasharo TrustRoot feature.
+
+!!! warning
+
+    This operation is irreversible and can seriously hinder the devices
+    usability for the sake of security. Make sure you understand the
```

Comment:
> and can seriously hinder the devices
    usability
    
What exactly? Which aspects of usability could be affected after fusing?